clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractFetch.java

```diff
@@ -783,9 +785,4 @@ public void close(final Timer timer) {
     public void close() {
         close(time.timer(0));
     }
-
-    private void requestMetadataUpdate(final TopicPartition topicPartition) {
```

Comment:
Author’s note: `requestMetadataUpdate` was moved to `FetchUtils` as it will be reused elsewhere in future commits.